config/samples/function-pubsub-sample-serving-only.yaml

```diff
@@ -46,21 +46,22 @@ spec:
   version: "v1.0.0"
   image: openfunctiondev/v1beta1-autoscaling-subscriber:latest
   serving:
-    runtime: "Async"
+    runtime: "async"
     scaleOptions:
       keda:
         scaledObject:
           pollingInterval: 15
           minReplicaCount: 0
           maxReplicaCount: 10
           cooldownPeriod: 30
-          triggers:
-            - type: kafka
-              metadata:
-                topic: "pubsub"
-                bootstrapServers: kafka-server-kafka-brokers.default.svc.cluster.local:9092
-                consumerGroup: autoscaling-subscriber
-                lagThreshold: "10"
+    triggers:
+      - type: kafka
+        metadata:
+          topic: "pubsub"
+          bootstrapServers: kafka-server-kafka-brokers.default.svc.cluster.local:9092
+          consumerGroup: autoscaling-subscriber
+          lagThreshold: "10"
+        targetKind: "object"
```

Comment:
Is it possible to add a default value for targetKind?
default to object if not set in the yaml?

Reply:
I've added the default value, like this: https://github.com/OpenFunction/OpenFunction/blob/69b4c94c41c14047f21e1741118c5e1fcd6de1c0/pkg/core/serving/openfuncasync/servingrun.go#L433-L442